### PR TITLE
Update documentation site to use sentence casing

### PR DESCRIPTION
**Pull request by phoebe-leung**
This PR fixes: https://github.com/StackExchange/Stacks/issues/323

Updated titles in Stacks documentation to use sentence casing.

### CODE REVIEW

File: docs/email/base/mso.html [hunk 1]
```diff
@@ -69,19 +69,19 @@
     </p>
 </section>
 <section class="stacks-section">
-    {% header h2 | Targeting Specific Outlook Versions %}
+    {% header h2 | Targeting specific outlook versions %}
```
Comment (TedGoas) [inline docs/email/base/mso.html]: *Outlook* should probably be capitalized here.

File: docs/product/guidelines/customization.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
 </section>
 
 <section class="stacks-section">
-    {% header h2 | Making Customizations %}
+    {% header h2 | Making customizations %}
```
Comment (TedGoas) [inline docs/product/guidelines/customization.html]: This page should have it's `<h3>'s` checked. Looks like the text for these is defined in this file:
`/docs/_data/product/configurations.yml`

File: docs/email/base/typography.html [hunk 3]
```diff
@@ -77,7 +77,7 @@ <h2 style="font-weight: bold; font-size: 21px; line-height: 24px; color: #242629
             <a href="https://stackoverflow.com/" style="color: #0077CC; text-decoration: none;">Go to the homepage</a>
         </div>
     </div>
-    {% header h3 | Links in Markdown %}
+    {% header h3 | Links in markdown %}
```
Comment (aaronshekey) [inline docs/email/base/typography.html]: Markdown should likely be capitalized here and in the following paragraph.

File: docs/email/components/team-identification.html [hunk 4]
```diff
@@ -49,4 +49,4 @@
             </table>
         </div>
     </div>
-</section>
\ No newline at end of file
+</section></section>
```
Comment (aaronshekey) [inline docs/email/components/team-identification.html]: Something happened here.

File: docs/email/templates/major-announcements.html [hunk 5]
```diff
@@ -36,4 +36,4 @@ <h3 class="my8">Teams Announcement</h3>
             <p class="fc-light">Sent May 2018</p>
         </a>
     </div>
-</section>
\ No newline at end of file
+</section></section>
```
Comment (aaronshekey) [inline docs/email/templates/major-announcements.html]: Double `</section>`

File: docs/product/base/current-color.html [hunk 6]
```diff
@@ -61,4 +61,4 @@
             </div>
         </div>
     </div>
-</section>
```
Comment (aaronshekey) [inline docs/product/base/current-color.html]: Another `</section>` addition

File: docs/product/components/tabs.html [hunk 7]
```diff
@@ -122,10 +122,10 @@
...
-    {% header h2 | Content Guidelines %}
+    {% header h2 | Content guidelines %}
     <p class="stacks-copy">Tabs should:</p>
     <ul class="stacks-list">
         <li class="">Tab labels should be short, distinct, and scannable. Ideally they are a word or two.</li>
         <li class="">Tab label naming should be consistent within a group. For example, if most labels within a verb, then all should start with a verb. If most labels describe a status, all labels should describe a status. If it is difficult to do so, reconsider if you are grouping similar content.</li>
     </ul>
-</section>
\ No newline at end of file
+</section></section>
```
Comment (aaronshekey) [inline docs/product/components/tabs.html]: Double `</section>`

**Pull request merged**